docs/README.md

```diff
@@ -1,5 +1,10 @@
 # docs
-Here we document important concepts and functionality formica supports.
+Here we document important concepts and functionality Inago supports.
 
 ### table of contents
-- [slice expansion](slice_expansion.md)
+
+- [What is the goal of Inago?](goal.md)
+- [What do we expect from the users?](expect.md)
```

Comment:
I don't understand this. Unit file content doesn't need to be in a certain format. The only requirement is the unit file names. 

Maybe we have a communication issue here. When I read `What do we expect from the users?`, I think `Don't be dumb`, but this is probably not what you mean. May you want to write this part of the documentation yourself like I already stated in my other comment about the fleet structure (which I also do not understand properly).